src/csvcubed/models/validationerror.py

```diff
@@ -69,4 +69,11 @@ class ValidateModelPropertiesError(ValidationError):
     This error will be returned with the name of the variable that has not been validated.
     """
 
-    property_name: str
+    property_path: List[str]
+    offending_value: Any
+
+    def __post_init__(self):
+        self.message = (
```

Comment:
you should use `self.message += ...` to make it a bit terser.

Reply:
Changed